app/utils/helpers.js

```diff
@@ -41,11 +41,19 @@ export const getFilterData = (data, existing = [], field = "id") => {
   return unique;
 };
 
+
+const getSavedSortParamsFromLocalStorage = (key) => {
+  let tableProp = localStorage.getItem(key + TABLE_PROPERTIES);
+  tableProp = JSON.parse(tableProp || null) || {};
```

Comment:
it will be nice to have a helper function like 
```suggestion
  tableProp = parseJson(tableProp, default={});
```
So that other places that do the same thing can benefit from this